server/services/participants.js

```diff
@@ -165,12 +166,18 @@ const updateParticipant = async (participantInfo) => {
       { history: participantInfo.history || [], userUpdatedAt: new Date().toJSON() }
     );
     if (changes.interested === 'withdrawn') {
+      // Prevent withdrawal of hired participants HCAP-1354
+      const participantHiredStatuses = await getParticipantHiredStatuses(participantInfo.id);
```

Comment:
This is what I thought I saw would prevent hiring archived participants. I'm not so sure anymore (I put in a console log here, it didn't log).